components/engine/src/util.rs

```diff
@@ -1,13 +1,17 @@
 // Copyright 2019 TiKV Project Authors. Licensed under Apache-2.0.
 
 use crate::rocks;
-use crate::rocks::{Writable, WriteBatch, DB};
+use crate::rocks::{Writable, DB};
 
 use super::Result;
 use super::{IterOption, Iterable};
 use engine_traits::CF_LOCK;
 use tikv_util::keybuilder::KeyBuilder;
 
+use crate::Mutable;
+use rocksdb::WriteBatch;
```

Comment:
Can we use engine_trait::WriteBatch here?

Reply:
No, it would require reimplementing at least two traits that are already implemented in engine_rocks (which can't be linked to this crate). I'd rather focus on moving this module into engine_traits, but in a future patch.